### PR TITLE
loosely type site section

### PR DESCRIPTION
**Pull request by jamesgorrie**
## Who is this for?
🗺 

## What is it doing for them?
Makes sure people know where they are on the site.

### CODE REVIEW

File: common/model/pages.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ export type Page = {|
   promo: ?ImagePromo,
   body: any[],
   datePublished: ?Date,
+  // This is just for now, we will be implementing a proper site tagging
```
Comment (davidpmccormick) [inline common/model/pages.js]: Could this have a `TODO:` or `FIXME:` for ease of searching for stuff we want to update?

Comment (jamesgorrie) [reply common/model/pages.js]: Quite liking the syntax of `TODO ({label})` syntax.

Comment (davidpmccormick) [reply common/model/pages.js]: I'll need to configure my 'Find TODOs' plugin accordingly (think it only searches for `TODO:` and `FIXME:` by default)

Comment (jamesgorrie) [reply common/model/pages.js]: ah, okay, I'll do it differently.
How about `TODO: (drupal migration) blah blah`?

**Pull request merged**